qcodes/instrument_drivers/QuTech/S5i.py

```diff
@@ -14,43 +14,49 @@ class S5i(Instrument):
 
     Args:
         name (str): name of the instrument.
-
         spi_rack (SPI_rack): instance of the SPI_rack class as defined in
             the spirack package. This class manages communication with the
             individual modules.
-
         module (int): module number as set on the hardware.
     """
 
     def __init__(self, name, spi_rack, module, **kwargs):
         super().__init__(name, **kwargs)
 
-        self.s5i = S5i_module(spi_rack, module)
+        self.s5i = S5i_module(spi_rack, module, frequency=41e6, enable_output=1,
+                              output_level=0)
```

Comment:
these kwargs resemble the parameters below. 

* Shouldn't the parameters below _immediately_ reflect these values?
* Shouldn't these values be settable upon instantiation of `S5i` class? perhaps, in some time, the frequency will change, or you will _not_ want to enable output at the initialization.

Please, i really beg, keep things parameterizable/settable/adjustible, so that when one will need to change `41e6` in `frequency=41e6`, he will _not_ need to make a new PR against qcodes.

Reply:
done